oktapam/resource_server_checkout_settings.go

```diff
@@ -28,7 +27,8 @@ func NewServerCheckoutSettingsResource() resource.Resource {
 }
 
 type serverCheckoutSettingsResource struct {
-	client *client.SDKClientWrapper
+	api      *pam.ProjectsAPIService
```

Comment:
We actually don't need the sdkclientwrapper.. we just need the projectsapi

Reply:
We used the SDK client wrapper here to add consistent error handling and manage common HTTP status codes in a standardized way, similar to the old client. Is there any reason to revert to using the SDK API directly?

Reply:
This resource makes no use of any of those things.